lib/runCode.js

```diff
@@ -200,8 +200,12 @@ module.exports = function (opts, cb) {
         // run the opcode
         var result = opFn.apply(null, args)
       } catch (e) {
-        cb(e)
-        return
+        if (e.errorType && e.errorType === 'VmError') {
```

Comment:
I think instead of `errorType` you could use something like `e is VmError`.

Reply:
Ah, no, tested this. Even on a VmError is still of common type Error.

Reply:
(this was my first idea as well)